src/pages/AddPersonalBankAccountPage.js

```diff
@@ -85,7 +85,7 @@ class AddPersonalBankAccountPage extends React.Component {
         const shouldShowSuccess = lodashGet(this.props, 'personalBankAccount.shouldShowSuccess', false);
 
         return (
-            <ScreenWrapper includeSafeAreaPaddingBottom={shouldShowSuccess}>
+            <ScreenWrapper includeSafeAreaPaddingBottom={shouldShowSuccess} shouldEnablePickerAvoiding={false}>
```

Comment:
Sorry to be a pain, but would you mind putting the props on newlines now that there are two? Thanks!

```
<ScreenWrapper
    includeSafeAreaPaddingBottom={shouldShowSuccess}
    shouldEnablePickerAvoiding={false}
>
```

Reply:
Done! (& merged main)